pages/docs/concepts/asyncapi-document/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Introduction'
+weight: 1
+---
+
+The AsyncAPI document is a YAML or JSON file that describes your implementation of the AsyncAPI specification. Providing a standardized way to document and describe asynchronous and event-driven systems, the AsyncAPI document generates documentation/code and defines the different components of your event-driven application, such as the events, channels, and message formats.
```

Comment:
> the AsyncAPI document generates documentation/code {...}

The document doesn't generate documentation or code per see, but through generators and other tooling. I would reword that to clarify it.

>  {...} and defines the different components of your event-driven application, such as the events, channels, and message formats.

I would replace the word `events` by `messages` because events is only one type of messages. For example, can be a request, a command, a query, etc.

Reply:
Oh got it, right! Fixed both.